goaviatrix/gateway.go

```diff
@@ -1161,3 +1173,71 @@ func (c *Client) DisableMonitorGatewaySubnets(gateway *Gateway) error {
 	}
 	return c.PostAPI(action, form, BasicCheck)
 }
+
+func (c *Client) EnableVPNConfig(gateway *Gateway, vpnConfig *VPNConfig) error {
+	action := "edit_vpn_config"
+	form := map[string]interface{}{
+		"CID":     c.CID,
+		"action":  action,
+		"command": "enable",
+		"vpc_id":  gateway.VpcID,
+		"lb_name": gateway.GwName,
+		"key":     vpnConfig.Name,
+		"value":   vpnConfig.Value,
+	}
+	return c.PostAPI(action, form, BasicCheck)
+}
+
+func (c *Client) DisableVPNConfig(gateway *Gateway, vpnConfig *VPNConfig) error {
+	action := "edit_vpn_config"
+	form := map[string]interface{}{
+		"CID":     c.CID,
+		"action":  action,
+		"command": "disable",
+		"vpc_id":  gateway.VpcID,
+		"lb_name": gateway.GwName,
+		"key":     vpnConfig.Name,
+		"value":   "600",
```

Comment:
Why "600" is needed here? Can we leave this out?

Reply:
API to disable the feature requires the value input. This value is hard code. The updated RP use "-1" as value input, which will be more consistent to our attribute design.